libs/component-library/src/lib/tabs/tabs.tsx

```diff
@@ -117,22 +212,22 @@ export const FxTabs = ({
   });
 
   return (
-    <FxBox
-      flexDirection="row"
-      onLayout={(event) => {
-        containerWidth.value = event.nativeEvent.layout.width;
-      }}
-    >
-      {items.map((item, idx) => {
-        return (
-          <FixedTab
-            text={item}
-            key={idx}
-            selected={idx === selectedIdx}
-            onPress={() => selectHandler(idx)}
-          />
-        );
-      })}
+    <FxBox flexDirection="row">
+      {variant === 'auto'
+        ? tabsItems.reduce((acc, cur) => {
+            if (!acc) {
+              return cur;
+            } else {
+              return (
+                <>
+                  {acc}
+                  <FxSpacer width={24} />
+                  {cur}
+                </>
+              );
+            }
+          })
+        : tabsItems}
```

Comment:
Not such a fan of this reduce pattern just to render the space for auto. Could `tabItems` not be updated as such:
```diff
const tabsItems = items.map((item, idx) => {
+    const isAuto = variant === 'auto';
+    const isLast = idx === items.length - 1;
+    const renderSpacer = isAuto && !isLast;
    return (
+      <>
        <View
          key={idx}
          style={styles[variant]}
          onLayout={(event) => {
            const { x, width } = event.nativeEvent.layout;
            itemLayouts.current[idx] = {
              x: x,
              width: width,
            };

            if (idx === selectedIdx) {
              selectedLayout.value = { x: x, width: width };
            }
          }}
        >
          <Tab
            text={item}
            selected={idx === selectedIdx}
            onPress={() => selectHandler(idx)}
            variant={variant}
          />
        </View>
+       {renderSpacer && <FxSpacer width={24} />}
+     </>
    );
  });
```

And then:
```diff
-{variant === 'auto'
-        ? tabsItems.reduce((acc, cur) => {
-            if (!acc) {
-              return cur;
-            } else {
-              return (
-                <>
-                  {acc}
-                  <FxSpacer width={24} />
-                  {cur}
-                </>
-              );
-            }
-          })
-        : tabsItems}
+        {tabsItems}
        ```